testcontainers-dapr/src/test/resources/components/statestore.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: dapr.io/v1alpha1
+kind: Component
+metadata:
+  name: statestore
+spec:
+  type: state.redis
+  version: v1
+  metadata:
+    - name: keyPrefix
+      value: name
+    - name: redisHost
+      value: redis:6379
+    - name: redisPassword
+      value: ""
```

Comment:
Could we add a new line to make GitHub happy 😄

Reply:
done